apps/src/code-studio/datasetLibrary.json

```diff
@@ -3,22 +3,26 @@
     {
       "name": "dogs",
       "description": "Different breeds of dogs, including images and rankings on different qualities like size and friendliness",
-      "url": "/api/v1/dataset-library/dogBreeds.csv"
+      "url": "/api/v1/dataset-library/dogBreeds.csv",
+      "columns": "name,size,dogFriendly,imgURL"
     },
     {
       "name": "words",
       "description": "The 1000 most commonly used English words and their parts of speech",
-      "url": "/api/v1/dataset-library/words.csv"
+      "url": "/api/v1/dataset-library/words.csv",
+      "columns": "word,partOfSpeech"
     },
     {
       "name": "countries",
       "description": "Information about different countries, images of their flags, and MP3 of their anthem",
-      "url": "/api/v1/dataset-library/countries.csv"
+      "url": "/api/v1/dataset-library/countries.csv",
```

Comment:
Nit - some of these are captialized, some not. Nit for consistency, but not ship-blocking.